crates/wasm-mutate/src/mutators/mod.rs

```diff
@@ -24,6 +25,11 @@ pub trait Mutator {
     }
 }
 
+// Helper type to return operator and ofsset inside the byte stream
+/// Type helper to wrap operator and the byte offset in the code section of a Wasm module
+pub type OperatorAndByteOffset<'a> = (Operator<'a>, usize);
```

Comment:
The `//` comment seems like a bad copy-paste from a function that returns `OperandAndByteOffset`s?